src/pages/Collaborators.tsx

```diff
@@ -0,0 +1,133 @@
+import { Header } from "../componets/Header/Header"
+import { SideBar } from "../componets/SideBar/SideBar"
+import { VfoodsContext } from "../contexts/VfoodsContext"
+import { collaboratorType } from "../types"
+import { useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import Api from "../Api";
+import starSvg from '../assets/starGray.svg';
+
+
+export default function Collaborators() {
+    const { allCollaborators } = useContext(VfoodsContext);
+    const [tableData, setTableData] = useState([] as { colaborador: collaboratorType, membro: string, status: number }[]);
+    let arrayNotas = [] as { colaborador: collaboratorType, nota: number }[];
+
+    const loadTableData = () => {
+        let newData = allCollaborators.map((collaborator) => {
+            const year = new Date(collaborator.data_admissao).getFullYear();
+            return {
+                colaborador: collaborator,
+                membro: `desde ${year}`,
+                status: arrayNotas.find((nota) => nota.colaborador.id === collaborator.id)?.nota || 0,
+            };
+        });
+
+        newData.sort((a, b) => b.status - a.status);
+
+        setTableData(newData);
+    };
+
+    async function loadNotas(colaboradores: collaboratorType[]): Promise<any> {
+        arrayNotas = [] as { colaborador: collaboratorType, nota: number }[];
+
+        for (let i = 0; i < colaboradores.length; i++) {
+            const currMonth = new Date().getMonth() + 1;
+            const currYear = new Date().getFullYear();
+            const url = `nota-mensal/${colaboradores[i].id}/${currMonth}/${currYear}`;
+
+            try {
+                const response = await Api.get(url);
+                const data = response.data;
+
+                arrayNotas.push({ colaborador: colaboradores[i], nota: data.notaMensal });
+            } catch (error) {
+                console.error(`Ocorreu um erro ao carregar a nota para o colaborador ${colaboradores[i].id}`);
+            }
+        }
+    };
+
+    const notaEstrelas = (nota: number) => {
+        let estrelas = [];
+        for (let i = 0; i < nota; i++) {
+            estrelas.push(<img src={starSvg} alt="Estrela" className="w-4 h-4" key={i} />);
+        }
+        if (nota === 0) {
+            estrelas.push(<p className='text-gray-500 text-sm'>Sem nota</p>);
```

Comment:
Essa nota 0 tu pensa caso a pessoa não tenha conseguido fazer nada mesmo ou caso ela n tem nada registrado? Pq se for que a pessoa "tirou" nota 0 eu sou a favor de deixar 1 estrela como mínimo.

Reply:
Era pra quando não tem nota registrada, esqueci que o cabra pode tirar 0 mesmo kkk vou ajustar isso